smartcontractkit/external-adapters-js
# Deprecate onchain adapter

## Closes 25880

## Description

The Data team has confirmed that we no longer need the Onchain adapter as it was used for the `BRENT-USD` feed, which is now dead.

## Changes

- Remove `onchain` EA
- Remove all references to it in `legos` and other generated files.



## Steps to Test

1. Run `yarn && yarn setup` to ensure that the project can still be built
2. Run any adapter and ensure that it is still working

## Quality Assurance
